server/lib/config/pbench-server-default.cfg

```diff
@@ -100,6 +100,11 @@ realm = pbench-server
 # Client entity name requesting OIDC to authenticate a user.
 client = pbench-client
 
+# Cert location for connecting to the OIDC client
+# Boolean value represent the use of system-wide certs. If you want to use a
+# custom CA then its location path should be recorded.
+cert_location = True
```

Comment:
I think these are always String values, so I'm not sure this'll work the way you seem to expect.

Reply:
Ah, that is true these are always string values, I have to convert it to boolean in the code.

Reply:
Right, which I why I suggested that we should just comment it out here and handle the not-defined case in the code.